### PR TITLE
Added: a FAQs section with certificate verification option

### PR DESCRIPTION
**Pull request by TheoMauri**
A FAQs has been added with one question about the certificate verification and its answer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,3 +91,48 @@ Note that the `request` method provided by this module always outputs a generato
...
+Certificate verification can be bypassed with the `verify_ssl` option.
+
+For example, this option was added to the previous ping.py script as follows:
+
+```python
+from cyberwatch_api import Cyberwatch_Pyhelper
+
+output = Cyberwatch_Pyhelper().request(
+    method="get",
+    endpoint="/api/v3/ping"
```
Comment (PierreHachard) [inline README.md]:
```suggestion
    endpoint="/api/v3/ping",
```

This cannot work, please test your code

File: README.md [hunk 2]
```diff
@@ -89,3 +91,48 @@ Note that the `request` method provided by this module always outputs a generato
...
+output = Cyberwatch_Pyhelper().request(
+    method="get",
+    endpoint="/api/v3/assets/servers/{id}",
+    params={'id' : 7}
+)
+```
+```python
+output = Cyberwatch_Pyhelper().request(
+    method="put",
+    endpoint="/api/v3/vulnerabilities/servers/{id}",
```
Comment (PierreHachard) [inline README.md:123]:
```suggestion
    endpoint="/api/v3/vulnerabilities/servers/7",
```

File: README.md [hunk 3]
```diff
@@ -89,3 +91,48 @@ Note that the `request` method provided by this module always outputs a generato
...
+    body_params={'id' : 7, 'description' : "this is a description", "groups":[3,4]}
```
Comment (PierreHachard) [inline README.md]:
```suggestion
    body_params={'description' : "this is a description", "groups":[3,4]}
```

As of today, passing the ID through body_params does not work, so remove it from the example and just pass the ID through the endpoint

File: README.md [hunk 4]
```diff
@@ -89,3 +91,48 @@ Note that the `request` method provided by this module always outputs a generato
...
+)
+```
+
+## FAQs
+
+**What if I don't want to verify the SSL certificate?**
+
+Error example:
```
Comment (PierreHachard) [inline README.md:132]:
```suggestion
Error example:

```

let's add a carriage return before code blocks to be consistent with the rest of the documentation

**Review by PierreHachard — COMMENTED** [review]
Please rebase locally also

File: README.md (hunk 3 above)
Comment (hezanathos) [reply README.md]: It has been fixed and released.

File: README.md [hunk 5]
```diff
@@ -120,6 +121,33 @@ output = Cyberwatch_Pyhelper().request(
 output = Cyberwatch_Pyhelper().request(
     method="put",
     endpoint="/api/v3/vulnerabilities/servers/{id}",
-    body_params={'id' : 7, 'description' : "this is a description", "groups":[3,4]}
+    body_params={'id':7,'description' : "this is a description", "groups":[3,4]}
```
Comment (hezanathos) [inline README.md]:
```suggestion
    body_params={'id' : 7,'description' : "this is a description", "groups":[3,4]}
```

**Pull request merged**